spec/flatware_helper.rb

```diff
@@ -1,4 +1,5 @@
 return if ENV["CI"]
+return if ENV["NOFW"]
```

Comment:
what's this for?

Reply:
Better to just add `return unless defined?(Flatware)`

Reply:
Doing this you can also just remove the `ENV` check